IPython/core/latex_symbols.py

```diff
@@ -102,68 +103,68 @@
     "\\aa" : "å",
     "\\ae" : "æ",
     "\\eth" : "ð",
+    "\\dh" : "ð",
     "\\o" : "ø",
     "\\th" : "þ",
     "\\DJ" : "Đ",
     "\\dj" : "đ",
-    "\\Elzxh" : "ħ",
     "\\imath" : "ı",
+    "\\jmath" : "ȷ",
     "\\L" : "Ł",
     "\\l" : "ł",
     "\\NG" : "Ŋ",
     "\\ng" : "ŋ",
     "\\OE" : "Œ",
     "\\oe" : "œ",
-    "\\texthvlig" : "ƕ",
-    "\\textnrleg" : "ƞ",
-    "\\textdoublepipe" : "ǂ",
-    "\\Elztrna" : "ɐ",
-    "\\Elztrnsa" : "ɒ",
-    "\\Elzopeno" : "ɔ",
-    "\\Elzrtld" : "ɖ",
-    "\\Elzschwa" : "ə",
-    "\\varepsilon" : "ɛ",
-    "\\Elzpgamma" : "ɣ",
-    "\\Elzpbgam" : "ɤ",
-    "\\Elztrnh" : "ɥ",
-    "\\Elzbtdl" : "ɬ",
-    "\\Elzrtll" : "ɭ",
-    "\\Elztrnm" : "ɯ",
-    "\\Elztrnmlr" : "ɰ",
-    "\\Elzltlmr" : "ɱ",
-    "\\Elzltln" : "ɲ",
-    "\\Elzrtln" : "ɳ",
-    "\\Elzclomeg" : "ɷ",
-    "\\textphi" : "ɸ",
-    "\\Elztrnr" : "ɹ",
-    "\\Elztrnrl" : "ɺ",
-    "\\Elzrttrnr" : "ɻ",
-    "\\Elzrl" : "ɼ",
-    "\\Elzrtlr" : "ɽ",
-    "\\Elzfhr" : "ɾ",
-    "\\Elzrtls" : "ʂ",
-    "\\Elzesh" : "ʃ",
-    "\\Elztrnt" : "ʇ",
-    "\\Elzrtlt" : "ʈ",
-    "\\Elzpupsil" : "ʊ",
-    "\\Elzpscrv" : "ʋ",
-    "\\Elzinvv" : "ʌ",
-    "\\Elzinvw" : "ʍ",
-    "\\Elztrny" : "ʎ",
-    "\\Elzrtlz" : "ʐ",
-    "\\Elzyogh" : "ʒ",
-    "\\Elzglst" : "ʔ",
-    "\\Elzreglst" : "ʕ",
-    "\\Elzinglst" : "ʖ",
-    "\\textturnk" : "ʞ",
-    "\\Elzdyogh" : "ʤ",
-    "\\Elztesh" : "ʧ",
+    "\\hvlig" : "ƕ",
+    "\\nrleg" : "ƞ",
+    "\\doublepipe" : "ǂ",
+    "\\trna" : "ɐ",
+    "\\trnsa" : "ɒ",
+    "\\openo" : "ɔ",
+    "\\rtld" : "ɖ",
+    "\\schwa" : "ə",
+    "\\varepsilon" : "ε",
```

Comment:
Regarding https://github.com/ipython/ipython/pull/11400#issuecomment-432867724 `\varepsilon` is still present.

Reply:
It is changed the Unicode codepoint, now it is the *lunate* version as explained in https://github.com/ipython/ipython/issues/11399#issuecomment-431696925

Reply:
Yes, I understand the point I was just doing and dumb listing of the changes from a user point of view, and it still lead to the `ε` character being untypable with the latex equivalent. You can still use `\Greek Small Letter Epsilon<tab>` but I want to properly document it. Appologies if it was a confusing comment